packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -796,12 +797,12 @@ export class Relayer {
     let deposit: Deposit | undefined = this.l2Client.getDepositByHash(relay.depositHash);
     if (deposit !== undefined) return deposit;
     // We could not find a deposit using the L2 client's default block search config. Next, we'll modify the block
-    // search config using the bridge pool's deployment time. This allows us to capture any deposits that happened
-    // outside of the L2 client's default block search config.
+    // search config using the bridge deposit box's deployment block. This allows us to capture any deposits that
+    // happened outside of the L2 client's default block search config.
     else {
       let blockSearchConfig = {
-        fromBlock: this.deployTimestamps[relay.l1Token].blockNumber,
-        toBlock: this.deployTimestamps[relay.l1Token].blockNumber + this.l2LookbackWindow,
+        fromBlock: this.l2DeployData[relay.chainId].blockNumber,
```

Comment:
This is the key bug fix. We should initiate this block search using L2 deployment data, not L1.